zglob_test.go

```diff
@@ -136,3 +136,34 @@ func TestMatch(t *testing.T) {
 		}
 	}
 }
+
+func TestFollowSymlinks(t *testing.T) {
+	tmpdir, err := ioutil.TempDir("", "zglob")
+	if err != nil {
+		t.Fatal(err)
+	}
+	os.MkdirAll(filepath.Join(tmpdir, "foo"), 0755)
+	ioutil.WriteFile(filepath.Join(tmpdir, "foo/baz.txt"), []byte{}, 0644)
+	os.Symlink(filepath.Join(tmpdir, "foo"), filepath.Join(tmpdir, "bar"))
+	defer os.RemoveAll(tmpdir)
```

Comment:
```go
	os.MkdirAll(filepath.Join(tmpdir, "foo"), 0755)
	ioutil.WriteFile(filepath.Join(tmpdir, "foo/baz.txt"), []byte{}, 0644)
	defer os.RemoveAll(tmpdir)

	err = os.Symlink(filepath.Join(tmpdir, "foo"), filepath.Join(tmpdir, "bar"))
	if err != nil {
		t.Skip(err.Error())
	}
```
Windows require privilege to make symlink.

Reply:
ah, makes sense, thanks for the snippet